mpc_follower_wrapper/src/mpc_follower_wrapper_worker.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2018-2020 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include "mpc_follower_wrapper/mpc_follower_wrapper_worker.hpp"
+
+namespace mpc_follower_wrapper {
+
+autoware_msgs::Waypoint MPCFollowerWrapperWorker::TrajectoryPlanPointToWaypointConverter(double current_time, geometry_msgs::PoseStamped pose, cav_msgs::TrajectoryPlanPoint tpp, cav_msgs::TrajectoryPlanPoint tpp2) {
```

Comment:
Something caught by sonar cloud is that this function does not use the current pose or current time. You should remove these arguments to avoid confusion.